### PR TITLE
grammatical errors corrected

### PR DESCRIPTION
**Pull request by AnmolxSingh**
Fixes the following PR:
fix spelling issues #701 

## Description of the changes
- Following changes have made to correct the grammatical mistake in the docs

### CODE REVIEW

File: content/docs/1.57/tools.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ weight: 12
 
 ## Tracegen
 
-It is a powerful utility that can be used to generate continuous stream of simple spans. This is specially useful for stress testing. It supports flags to control the number of workers and number of traces to generate in each worker. Not only that, it can generate spans in firehose mode (a flag set on spans to skip indexing).
+It is a powerful utility that can be used to generate a continuous stream of simple spans. This is specially useful for stress testing. It supports flags to control the number of workers and number of traces to generate in each worker. Not only that, it can generate spans in firehose mode (a flag set on spans to skip indexing).
```
Comment (yurishkuro) [inline content/docs/1.57/tools.md:10]: I suggest instead changing this in the `next-release/*.md`  files only

Comment (AnmolxSingh) [reply content/docs/1.57/tools.md:10]: Alright i will be commiting changes to the next-release/*.md files

**Comment by AnmolxSingh** [comment]
changes committed to correct grammatical errors from next-release/*.md files

**Review by yurishkuro — APPROVED** [review]

**Comment by yurishkuro** [comment]
Please sign the commits

**Comment by AnmolxSingh** [comment]
All the commits have been signed

**Comment by yurishkuro** [comment]
the DCO check is still failing. Easiest is to squash all commits into and and sign it.

**Comment by AnmolxSingh** [comment]
Again some checks are not passing i think i should close this and open a new PR

**Pull request closed without merging**